api-catalog-ui/frontend/src/reducers/authentication.reducer.jsx

```diff
@@ -1,12 +1,17 @@
 import userConstants from '../constants/user.constants';
 
-function authenticationReducer(state = {}, action) {
+const sessionDefaultState = {
+    sessionOn: false,
+}
+
+function authenticationReducer(state=sessionDefaultState, action) {
```

Comment:
I got this notification:
Move parameter "state" after parameters without default value.sonarlint(javascript:S1788)

Reply:
Where did you see it? I can't find it

Reply:
it's from vs code

Reply:
hmm IntelliJ doesn't display it. I don't understand it